maquetado/scss/book-list.scss

```diff
@@ -31,3 +31,21 @@
   margin-bottom: 8px;
   margin-right: 33px;
 }
+
+.logo {
+  width: 212.42px;
+  height: 100%;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  position: fixed;
+  top: 0;
+  border-top: 6px solid $blue;
+  box-shadow: 1px 0px 5px;
+  padding: 0 246px;
+  background-color: white;
+}
```

Comment:
Remember to chain your PRs so I can see only the corresponding changes to each one of them.